### PR TITLE
Feature/#77 obfuscation #134 #135 #136

### LINKED ISSUE #77: Enable obfuscation by default.
https://flutter.dev/docs/deployment/obfuscate

don't forget the symbols file

### LINKED ISSUE #134: Version bump min iOS version to 12
Set the min iOS version to our default one

### LINKED ISSUE #135: Add config for Visual Studio Code
VSCode can run default flutter projects, but since we added flavors and different main files, there is some config needed.

### LINKED ISSUE #136: Flutter V1.22.6
Newest flutter version since 25 Jan 2020

### PR DESCRIPTION
**Pull request by ikbendewilliam**
Fixed #77 
Fixed #134 
Fixed #135 
Fixed #136

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 source "https://rubygems.org"
 
+gem 'cocoapods'
```
Comment (ikbendewilliam) [inline Gemfile:7]: This was necessary to run fastlane locally. Not sure if this is needed when building on server

File: ios/Podfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '9.0'
+platform :ios, '9.0'
```
Comment (vanlooverenkoen) [inline ios/Podfile:2]: aah it was this line you were refering to. Not sure if this is needed.

**Comment by vanlooverenkoen** [comment]
Maybe one general question. Maybe we should add something in the readme.md that obfuscation via fastlane is enabled by default and that the files are stored `./build/debug-info/#{options[:flavor]}` and maybe for jenkins we could add an extra info line what regex should be used for archivig these symbol files.

**Review by vanlooverenkoen — CHANGES_REQUESTED** [review]
Maybe we should add extra documentation to the readme.md that obfuscation is enabled by default when using fastlane for building. And that the symbol files are stored here `./build/debug-info/#{options[:flavor]}` 

an extra line could be a regex that can be used in jenkins for archiving these symbol files

File: ios/Podfile (hunk 2 above)
Comment (ikbendewilliam) [reply ios/Podfile]: Nope :-) doesn't seem so

**Review by vanlooverenkoen — APPROVED** [review]

**Pull request merged**